requirements.txt

```diff
@@ -1,3 +1,5 @@
+WHY TWO REQUIREMENT SETUPS?
```

Comment:
I followed marina mele tutorial for env setup(which had a folder for separate requirements files for development, production etc.) but heroku required a separate requirements.txt file, I can remove the requirements folder and just keep the requirements.txt file